main/src/io/github/iltotore/iron/RefinedTypeOps.scala

```diff
@@ -0,0 +1,18 @@
+package io.github.iltotore.iron
+
+type RefinedTypeOps[T] = T match
```

Comment:
Can you document (using the Scaladoc format) these types and methods?

Reply:
Sure, as far as I understand guidelines should be updated

Reply:
Yes I was thinking about Scaladoc comments but a new entry in the "static website" is also a good idea. You can do it if you have time but don't worry if you are too busy I'll still merge the PR.